### PR TITLE
Add an example.

### PR DESCRIPTION
**Pull request by kfox1111**
/kind documentation
```release-note
NONE
```

### CODE REVIEW

File: examples/pod.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    - 'while true; do sleep 1000; done'
+    volumeMounts:
+    - mountPath: /data
+      name: data
+  volumes:
+  - name: data
+    csi:
+      driver: image.csi.k8s.io
+      volumeAttributes:
+        image: kfox1111/misc:test
```
Comment (msau42) [inline examples/pod.yaml]: Should we use some public/common image here?

Comment (kfox1111) [reply examples/pod.yaml]: Yeah, I was wondering about that too...

What should we do? I could add an example image in the pr and add it to the release scripts too. Would we need a second docker repo for that to push to?

Comment (msau42) [reply examples/pod.yaml]: Would some popular image like nginx be sufficient? It's not really meant for this purpose but I don't think it's worth creating another repo for an example image

Comment (kfox1111) [reply examples/pod.yaml]: I think this issue is resolved for now. Can we merge this pr?

**Pull request merged**